src/envo/envo-edit.owl

```diff
@@ -39756,6 +39758,29 @@ AnnotationAssertion(Annotation(rdfs:comment "In German \"Kunststoff\" connotes t
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105101> "plastic"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105101> <http://purl.obolibrary.org/obo/ENVO_00010483>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105107> (marine anthropogenic litter)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105107> "Anthropogenic litter which is present in the marine or coastal environment."@en)
```

Comment:
Here, you should be more strict: if it's marine litter, then the definition should note "marine environments" without introducing coastal environments. You may want to create another class for "coastal anthropocentric litter"